### DISCUSSION TITLE
v10.0.0

### DISCUSSION THREAD

**Discussion by s34m** [topgrade-rs/topgrade#49] [Announcements]
## What's Changed
* Fix broken fisher update command by @tennox in https://github.com/topgrade-rs/topgrade/pull/33
* Revert fix for older fisher version by @tennox in https://github.com/topgrade-rs/topgrade/pull/37
* Added build scripts for multiple platforms by @DottoDev in https://github.com/topgrade-rs/topgrade/pull/47
* Changed binary name to be backwards compatible, version bump to 10.0.0 by @DottoDev in https://github.com/topgrade-rs/topgrade/pull/48

## New Contributors
* @tennox made their first contribution in https://github.com/topgrade-rs/topgrade/pull/33

**Full Changelog**: https://github.com/topgrade-rs/topgrade/compare/v9.1.0...v10.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/topgrade-rs/topgrade/releases/tag/v10.0.0'>v10.0.0</a>.</em>

**Comment by tennox** [comment]
Best first changelog appearance ever 😅

**Reply by s34m** [reply]
xD

**Comment by guihkx** [comment]
@DottoDev: Any plans to upgrade the [AUR package](https://aur.archlinux.org/packages/topgrade) as well? It seems like you were added as a co-maintainer.

Thanks.

**Reply by s34m** [reply]
Haven't noticed that xD.
r-darwish never told me that.
Thanks for the information will update that as soon as I have time
